src/components/picture/index.stories.mdx

```diff
@@ -0,0 +1,156 @@
+import { Meta, Story, Canvas, ArgsTable } from "@storybook/addon-docs";
+
+import Picture from ".";
+import Source from "./_children/source";
+import Image from "../image";
```

Comment:
Should not be importing the `Image` component here

Reply:
@matthewroach : Thank you! Fixed.